cli/mender-artifact/copy.go

```diff
@@ -21,18 +21,23 @@ import (
 	"regexp"
 
 	"github.com/urfave/cli"
+	"github.com/mendersoftware/mender-artifact/artifact"
 )
 
 var isimg = regexp.MustCompile(`\.(mender|sdimg|uefiimg)`)
 
 func Cat(c *cli.Context) (err error) {
+	comp, err := artifact.NewCompressorFromId(c.GlobalString("compression"))
+	if err != nil {
+		return cli.NewExitError("compressor '"+c.GlobalString("compression")+"' is not supported", 1)
```

Comment:
Always a good idea to include the original error, aka `...is not supported: "+err.Error()`.